lib/tasks/delius_trigger.rake

```diff
@@ -0,0 +1,14 @@
+namespace :delius do
```

Comment:
Is this on the correct branch?

Reply:
No you're right it belongs on the nDelius branch...